src/components/GlobalPreferences/HelpAndFeedback/HelpAndFeedback.js

```diff
@@ -74,16 +84,33 @@ function HelpAndFeedback({ historyPush, onClose, locator }) {
           css={`
             display: flex;
             flex-direction: column;
+            justify-content: center;
           `}
         >
-          <Button
+          <label
             css={`
-              margin-bottom: ${GU}px;
+              cursor: pointer;
+              display: flex;
+              align-items: center;
+              margin-bottom: ${2 * GU}px;
             `}
-            label="Go to the embedded console"
-            mode="strong"
-            onClick={handleGoToEmbeddedConsole}
-          />
+          >
+            <Switch
+              onChange={handleOptOutConsoleChange}
+              checked={!optedOutFromConsole}
+            />
+            <span
+              css={`
+                color: ${theme.surfaceContentSecondary};
+                padding-left: ${1.5 * GU}px;
+                align-items: center;
+                ${textStyle('title4')}
```

Comment:
From the design, I think this and the label in the HelpScout section should both be `'body2'`.